Add line showing how to use rbenv

lfdebrux opened the pull request:
It took me a little bit of digging to figure out how to use `rbenv`, so I wanted to add it to the manual to make life easier in future.

brenetic commented:
`rbenv rehash` should be run automatically when a gem is installed, so technically we shouldn't need to run it manually.

We should be specific about the ruby version since we are explicitly stating it in our .ruby_version file

`rbenv install 2.3.1`

Now that I've read the install bit through I realise it is a bit misleading. If we're telling the user to use rbenv we shouldn't tell them to use Homebrew to install ruby.

Let's have a chat. If you had to use this README to install ruby from scratch it would be cool to get more of your feedback.

lfdebrux commented:
@brenetic I've updated the installation section based on your comments.

I also removed the part about issues with Nokogiri with native Mac Ruby, as if we're telling people to use rbenv it doesn't really seem relevant.
